packages/radix-ui-themes/src/components/radio-card-group.css

```diff
@@ -0,0 +1,132 @@
+.rt-RadioCardGroupRoot {
+  user-select: none;
+  cursor: default;
+}
+
+.rt-RadioCardGroupItem {
+  display: flex;
+  align-items: center;
+  cursor: var(--cursor-radio);
+}
+
+/* Avoid unintentional drag interactions (e.g. on images) */
+.rt-RadioCardGroupItem * {
+  pointer-events: none;
+}
+
+/***************************************************************************************************
+ *                                                                                                 *
+ * SIZES                                                                                           *
+ *                                                                                                 *
+ ***************************************************************************************************/
+
+.rt-RadioCardGroupItem {
+  padding: var(--radio-card-group-item-padding);
+  border-radius: var(--radio-card-group-item-radius);
+  gap: var(--radio-card-group-item-gap);
+}
+
+@breakpoints {
+  .rt-RadioCardGroupRoot {
+    &:where(.rt-r-size-1) {
+      --radio-card-group-item-padding: var(--space-3);
+      --radio-card-group-item-radius: var(--radius-4);
+      --radio-card-group-item-gap: var(--space-2);
+    }
+    &:where(.rt-r-size-2) {
+      --radio-card-group-item-padding: var(--space-4);
+      --radio-card-group-item-radius: var(--radius-4);
+      --radio-card-group-item-gap: var(--space-3);
+    }
+    &:where(.rt-r-size-3) {
+      --radio-card-group-item-padding: var(--space-5);
+      --radio-card-group-item-radius: var(--radius-5);
+      --radio-card-group-item-gap: var(--space-4);
+    }
+  }
+}
+
+/***************************************************************************************************
+ *                                                                                                 *
+ * VARIANTS                                                                                        *
+ *                                                                                                 *
+ ***************************************************************************************************/
+
+/* surface */
+
+.rt-RadioCardGroupRoot:where(.rt-variant-surface) {
+  :where(.rt-RadioCardGroupItem) {
+    background-color: var(--color-surface);
+    box-shadow: inset 0 0 0 1px var(--gray-a5);
+
+    &:where(:hover) {
+      box-shadow: inset 0 0 0 1px var(--gray-a7);
+    }
+
+    &:where([data-state='checked']) {
+      outline: 2px solid var(--accent-9);
+      outline-offset: -1px;
+    }
+
+    &:where(:focus-visible) {
+      outline: 2px solid var(--accent-9);
+      outline-offset: -1px;
+      box-shadow: 0 0 0 4px var(--accent-a6);
```

Comment:
I used a lower step here for more contrast (compared to the WorkDS one)
I think it works better.

I'm also generally questioning whether this style is fine or not, we don't really have it anywhere else in Radix Themes.